src/lib.rs

```diff
@@ -74,14 +74,14 @@ pub enum Mpu6050Error<E> {
 }
 
 /// Handles all operations on/with Mpu6050
-pub struct Mpu6050<I> {
+pub struct Mpu6050<I, const WHO_AM_I_VALUE: u8 = DEFAULT_SLAVE_ADDR> {
```

Comment:
i think this is wrong, it should probably be:

```suggestion
pub struct Mpu6050<I, const WHO_AM_I_VALUE: u8 = WHOAMI_VALUE_MPU6886> {
```